src/colorfulness.jl

```diff
@@ -0,0 +1,57 @@
+using ImageCore: GenericGrayImage
+using Statistics: mean, std
```

Comment:
Can you move these to `ImageQualityIndexes.jl`? they're partially duplicated.

Reply:
Agreed